lib/bedrock.js

```diff
@@ -273,44 +233,7 @@ api.runOnce = function(id, fn, options, callback) {
  * @returns {Promise} Resolves when the operation completes.
  */
 api.runOnceAsync = async (id, fn, options = {}) => {
-  const promise = new Promise((resolve, reject) => {
-    // listen to run function once
-    process.on('message', _runOnce);
-
-    async function _runOnce(msg) {
-      // ignore other messages
-      if(!(_isMessageType(msg, 'bedrock.runOnceAsync') && msg.id === id)) {
-        return;
-      }
-
-      process.removeListener('bedrock.runOnceAsync', _runOnce);
-
-      if(msg.error) {
-        return reject(errio.fromObject(msg.error, {stack: true}));
-      }
-
-      if(msg.done) {
-        return resolve();
-      }
-
-      // run in this worker
-      const _msg = {
-        type: 'bedrock.runOnceAsync',
-        id,
-        done: true
-      };
-      try {
-        await fn();
-      } catch(e) {
-        _msg.error = errio.toObject(e, {stack: true});
-      }
-      process.send(_msg);
-    } // end _runOnce
-  });
-
-  // schedule function to run
-  process.send({type: 'bedrock.runOnceAsync', id, options});
-  return promise;
+  return _runOnce({type: 'bedrock.runOnceAsync', id, fn, options});
```

Comment:
SUPER NIT, No curly braces required around this.

Reply:
I think it's cleaner and easier to maintain to have names vs. just using ordered params.

Reply:
I meant around this whole function, it could be `(...) => _runOnce(...)`

Reply:
Oh! Ok, changed it.